crates/matrix-sdk-base/src/client.rs

```diff
@@ -430,8 +430,8 @@ impl BaseClient {
                 Ok(e) => e,
                 Err(e) => {
                     warn!(
-                        "Couldn't deserialize state event for room {}: {:?} {:#?}",
-                        room_id, e, raw_event
+                        "Couldn't deserialize state event for room {room_id}: \
+                         {e:?} {raw_event:#?}",
```

Comment:
Perhaps we shouldn't log the event here.

```suggestion
                        room_id = %room_id,
                        error = ?e,
                        "Couldn't deserialize state event"
```